### PR TITLE
Improve maven plugin tags documentation

### PR DESCRIPTION
**Pull request by shekharAggarwal**
As mentioned in https://github.com/spring-projects/spring-boot/issues/33528, I have updated the description of `tags`.

### CODE REVIEW

File: spring-boot-project/spring-boot-tools/spring-boot-maven-plugin/src/docs/asciidoc/packaging-oci-image.adoc [hunk 1]
```diff
@@ -190,7 +190,7 @@ The value supplied will be passed unvalidated to Docker when creating the builde
 | `false`
 
 | `tags`
-| One or more additional tags to apply to the generated image.
+|One or more additional tags to apply to the generated image. The values provided to the `tags` option should be full image references in the form of `[image name]:[tag]` or `[repository]/[image name]:[tag]`. The name `tags` was chosen to be consistent with the https://buildpacks.io/docs/tools/pack/#pack-cli[`pack CLI`].
```
Comment (scottfrederick) [inline spring-boot-project/spring-boot-tools/spring-boot-maven-plugin/src/docs/asciidoc/packaging-oci-image.adoc:193]: We can remove the last sentence `The name 'tags' was chosen to be consistent with the https://buildpacks.io/docs/tools/pack/#pack-cli['pack CLI']`. While accurate, the statement is true of many arguments here, and we don't need to call this one out specifically.

**Comment by scottfrederick** [comment]
Thanks for the contribution @shekharAggarwal. We need to make the same change to the [Gradle plugin documentation](https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot-tools/spring-boot-gradle-plugin/src/docs/asciidoc/packaging-oci-image.adoc). Are you interested in making that change also? 

If so, you can just push an additional commit to this branch and it will update the pull request.

**Comment by shekharAggarwal** [comment]
Sure

**Review by shekharAggarwal — COMMENTED** [review]
Removed unwanted statements.

**Comment by philwebb** [comment]
Thanks!

**Pull request closed without merging**